README.md

```diff
@@ -7,37 +7,29 @@ This project is useful as it has the basic functionalities of any normal calenda
 
 
 ## Development Setup
-It is recommended that you use VsCode when contributing to this project. Please install the Eslint and Prettier extensions so that the code style matches what has already been done before.
+It is recommended that you use **VsCode** when contributing to this project. Please install the **Eslint** and **Prettier** extensions so that the code style matches what has already been done before.
 
 ## Running the code
-We are using Npm for this particular project (https://www.npmjs.com/get-npm).
-```bash
-Npm install
-```
-```bash
-Npm start 
-```
+We are using npm for this particular project (https://www.npmjs.com/get-npm).
+### `cd doto-frontend/backend`
+(change to either frontend or backend depending on where you wish to contribute)
 
-Please check the front end readme for more information on running the front end of the code.
+### `npm install`
+
+### `npm start`
```

Comment:
Is there a reason for making the code headers.

Reply:
Um, It looks nice I guess. If you really don't want it then I can take it out